packages/ember/tailwind-webpack/index.js

```diff
@@ -0,0 +1,52 @@
+// @ts-check
+import { files, html, packageJson, project } from 'ember-apply';
+import path, { dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+// @ts-ignore
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export default async function run() {
+  await packageJson.addDevDependencies({
+    'tailwindcss': '^3.0.0',
+    'postcss': '^8.0.0',
+    'postcss-loader': '^7.0.0',
+    'autoprefixer': '^10.0.0'
+  });
+
+  // TODO: For some reason the css file in the [official guides](https://tailwindcss.com/docs/guides/emberjs) goes to `app/app.css` instead of `app/styles/app.css`. Do we want to honour that?
```

Comment:
> app/styles/app.css

this is the classic style location, and we basically ignore all of `app/styles` when we do modern CSS, because the old way of managing CSS is too hard to work with.